src/screens/OrganizationPeople/OrganizationPeople.module.css

```diff
@@ -208,17 +199,94 @@
     box-shadow 0.2s;
   width: 100%;
 }
-.sidebarsticky > input {
-  text-decoration: none;
-  margin-bottom: 50px;
-  border-color: #e8e5e5;
-  width: 80%;
-  border-radius: 7px;
-  padding-top: 5px;
-  padding-bottom: 5px;
-  padding-right: 10px;
-  padding-left: 10px;
-  box-shadow: none;
+.dropdown {
+  background-color: white;
+  border: 1px solid #31bb6b;
+  position: relative;
+  display: inline-block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  color: #31bb6b;
+}
+.input {
+  flex: 1;
+  position: relative;
+}
+/* .btnsContainer {
+  display: flex;
+  margin: 2.5rem 0 2.5rem 0;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+} */
+
+.btnsContainer {
+  display: flex;
+  margin: 2.5rem 0 2.5rem 0;
+}
+
+.btnsContainer .btnsBlock {
+  display: flex;
+}
+
+.btnsContainer .btnsBlock button {
+  margin-left: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.btnsContainer .input {
+  flex: 1;
+  position: relative;
+}
+
+.btnsContainer input {
+  outline: 1px solid var(--bs-gray-400);
+}
+
+.btnsContainer .input button {
+  width: 52px;
+}
+.searchBtn {
+  margin-bottom: 10px;
+}
+
+.inputField {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  background-color: white;
+  box-shadow: 0 1px 1px #31bb6b;
+}
+.inputField > button {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+.TableImage {
+  background-color: #31bb6b !important;
+  width: 50px !important;
+  height: 50px !important;
+  border-radius: 100% !important;
+  margin-right: 10px !important;
+}
+.tableHead {
+  background-color: #31bb6b !important;
+  color: white;
+  border-radius: 20px !important;
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.tableHead :nth-first-child() {
+  border-top-left-radius: 20px;
+}
+
+.mainpageright > hr {
```

Comment:
Use a single line to add all the margins

Reply:
yeah sure